R/brmsfit-helpers.R

```diff
@@ -255,7 +255,9 @@ get_cov_matrix_ac <- function(prep, obs = NULL, latent = FALSE) {
   }
   # prepare residual standard deviations
   if (latent) {
-    sigma2 <- as.numeric(prep$ac$sderr)^2
+    if (!is.null(prep$ac$sderr)) {
```

Comment:
Why should `prep$ac$sderr` ever be NULL in this case? Then sigma would not be defined at all?